app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Users
+  class OmniauthCallbacksController < Devise::OmniauthCallbacksController
```

Comment:
Do we really need all these generated controllers? As I remember if you do not make any changes to it - you could drop them.
And by the way here is the inconsistency with options you declare in user model.